<?xml version="1.0"?>
<launch>
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find planner_node)/config/exp.rviz"/>

  <node pkg="planner_node" type="planner_node1" name="planner_node1" output="screen">
    <rosparam command="load" file="$(find planner_node)/config/planner.yaml" ns="planning"/>
    <param name="planning/env_file" value="$(find planner_node)/config/env.json" />
  </node>
</launch>
